blackjax/sgmcmc/csgld.py

```diff
@@ -216,11 +228,12 @@ def init_fn(position: PyTree):
 
         def step_fn(
             rng_key: PRNGKey,
-            state,
+            state: ContourSGLDState,
             minibatch: PyTree,
             step_size_diff: float,
             step_size_stoch: float,
-        ):
+            temperature: float = 1.0,
```

Comment:
In Appendix J of https://arxiv.org/pdf/1902.03932.pdf they do some analysis on the robustness of the temperature, concluding that "We find that the optimal temperature is often less than 1. We hypothesize that this result is due to the model misspecification common to neural networks." For CIFAR-10 the optimal temperature is 0.1 and for CIFAR-100 it is 0.01. So I guess this is why they chose it, but not sure what would be the best default value in this case.

Reply:
I see ok, yes I think for our use case a default value of 1. makes more sense since we do not assume the model context never mind any cold posterior effect, and this is in line with the other `sgmcmc` algorithms